content/docs/state-and-lifecycle.md

```diff
@@ -31,16 +31,16 @@ setInterval(tick, 1000);
 
 [**Try it on CodePen**](https://codepen.io/gaearon/pen/gwoJZk?editors=0010)
 
-In this section, we will learn how to make the `Clock` component truly reusable and encapsulated. It will set up its own timer and update itself every second.
+इस खंड में, हम सीखेंगे कि `Clock` कौम्पोनॅन्ट को वास्तव में पुन: प्रयोज्य और समझाया कैसे बनाया जाए। यह अपना खुद का टाइमर सेट करेगा और हर सेकंड खुद को अपडेट करेगा।
```

Comment:
encapsulated -> समझाया doesn't seem right

Reply:
संपुटित still does not feel right, can we convert it to hindi straight up instead of translating?